### PR TITLE
Add ssl support

### PR DESCRIPTION
**Pull request by eado**
Issue #35 should be resolved.

### CODE REVIEW

**Comment by eado** [comment]
Documentation for the new features should be available in the updated README.md.

**Comment by keisuke4270131** [comment]
Thank you for your ssl support. It worked very well. And here is a simple example code for websocket client.

```
from websocket import create_connection
import ssl
ws = create_connection("wss://127.0.0.1:13254/", sslopt={"cert_reqs": ssl.CERT_NONE})
receive_message = ws.recv()
print(receive_message)
```

**Comment by Haschtl** [comment]
I get an error when a client tries to connect with the server: 

ssl.SSLError: [SSL: SSLV3_ALERT_CERTIFICATE_UNKNOWN] sslv3 alert certificate unknown (_ssl.c:777)

After changing the wrapping to get more information:
```
try:
     socket = ssl.wrap_socket(socket, server_side=True, certfile=server.cert, keyfile=server.key)
except Exception as e: # Not sure which exception it throws if the key/cert isn't found
     tb = traceback.format_exc()
     logging.warn(tb)
     logging.warn(e)
     logger.warn("SSL not available (are the paths {} and {} correct for the key and cert?)".format(server.key, server.cert))

```

I think something is wrong with my cert and/or key.
I'm creating the SSL-cert with
`openssl req -new -x509 -days 365 -nodes -out cert.pem -keyout key.pem` 
or 
`openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes`

both with the same error. What is the correct way to generate key and certificate?

File: websocket_server/websocket_server.py [hunk 1]
```diff
@@ -112,16 +113,22 @@ class WebsocketServer(ThreadingMixIn, TCPServer, API):
                 }
     """
 
+    key = None
+    cert = None
```
Comment (Pithikos) [inline websocket_server/websocket_server.py:117]: I think these are probably redundant? Since already set in the `__init__`?

**Pull request merged**